src/custom/utils/operator.ts

```diff
@@ -37,8 +37,19 @@ const DEFAULT_HEADERS = {
  */
 export type OrderID = string
 
-export interface OrderPostError {
-  errorType: 'MissingOrderData' | 'InvalidSignature' | 'DuplicateOrder' | 'InsufficientFunds'
+export enum ApiErrorCodes {
```

Comment:
Can you add a link to the swagger API (to the line where they define it), and mention that this should be in sync to it?

This way is easier to check from time to time if there's new errors to handle